CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project tries to adhere to [Semantic Versioning](http://semver.org/).
 
 ## [2.0.1] - UNRELEASED
+### Changed [JDR - ODC Space] - 2.0.1.dev1
```

Comment:
I think this breaks the formatting expected by http://keepachangelog.com/, mention the contributor within the list of things changes instead of in the heading. We might as well add an Authors markdown file to this PR also.